SmartDeviceLink/SDLProtocolListener.h

```diff
@@ -23,6 +23,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)onProtocolOpened;
 - (void)onProtocolClosed;
 - (void)onError:(NSString *)info exception:(NSException *)e;
+- (void)onTransportError:(NSError *)error;
```

Comment:
The "onError:exception:" delegate (one line above) isn't and hasn't been used anywhere in the project. We might want to remove it to avoid confusion.